x-pack/test/api_integration/apis/monitoring/logs/cluster.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
```

Comment:
Need to merge master and rewrite this import before merging: https://github.com/elastic/kibana/pull/33761